assists/subsystem.py

```diff
@@ -216,60 +216,12 @@ def set_cdo_flags(flags, dev_node, dev_handle):
 
     return access_list
 
-def cdo_requirement_access_expand( tree, subnode, verbose , access_chunks ):
```

Comment:
So what's the overall change to access_expand here ? The standard access_expand is called from some simple domain processing, since that is part of the generic spec.

We should document the high level changes in the commit message itself, and indicate if the result is different for existing callers.

Reply:
hi bruce,

im changing it right so there will be no more xilinx specific handling in the subsystem assist :)

ill update the pull request today